code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -212,15 +212,44 @@
 						)
 
 /obj/item/gun/ballistic/shotgun/automatic/combat/compact
-	name = "compact combat shotgun"
-	desc = "A compact version of the semi automatic combat shotgun. For close encounters."
+	name = "warden's combat shotgun"
+	desc = "A modified version of the semi automatic combat shotgun with a collapsible stock. For close encounters."
 	icon_state = "cshotgunc"
-	mag_type = /obj/item/ammo_box/magazine/internal/shot/com/compact
+	mag_type = /obj/item/ammo_box/magazine/internal/shot/com
 	w_class = WEIGHT_CLASS_NORMAL
-	unique_reskin = list("Tatical" = "cshotgunc",
-						"Slick" = "cshotgunc_slick"
+	var/stock = FALSE
+	recoil = 5
+	spread = 2
+	unique_reskin = list("Unfolded" = "cshotgun",
```

Comment:
Shotgun slick is the TG sprite shotgun

Reply:
No it isn't Trilby. Shotgun Slick is the hunting style shotgun.

Reply:
cshotgun is the combat shotgun sprite, as intended.

Reply:
They spawn as tatical. Quit lieing about the hunting style. I coded the reskins I know what is what